index.js

```diff
@@ -796,6 +796,9 @@
   Nothing.prototype.toString = def('Nothing#toString', [],
                                    R.always('Nothing()'));
 
+  //  Nothing#inspect :: Maybe a ~> String
+  Nothing.prototype.inspect = R.nAry(0, Nothing.prototype.toString);
+
```

Comment:
Two questions:

  - Is there a reason to use `R.nAry(0)` here rather than simply assign `Nothing#toString` to `Nothing#inspect`?

  - Rather than define `Nothing#inspect` and `Just#inspect`, could we define `Maybe#inspect`?

    ```javascript
    Maybe.prototype.inspect = def('Maybe#inspect', [], R.invoker(0, 'toString'));
    ```